core/src/test/scala/akka/persistence/jdbc/journal/JdbcJournalPerfSpec.scala

```diff
@@ -46,7 +51,7 @@ abstract class JdbcJournalPerfSpec(config: Config, schemaType: SchemaType)
   lazy val db = SlickExtension(system).database(cfg).database
 
   override def beforeAll(): Unit = {
-    dropCreate(schemaType)
+    Await.result(SchemaUtils.dropAndCreate(system), 2.seconds)
```

Comment:
.futureValue is available because `ScalaFutures` but perhaps we don't want the patience config for that here?

Reply:
that's how we did everywhere in akka-projections.

Reply:
I went dig into akka-projections to remember why we did so. Here the comment we have there

```scala
// don't use futureValue (patience) here because it can take a while to start the test container
```

If I remember correctly, creating the DB can take much longer than other async in tests and we didn't want to increase the patience config because of that. So, for the specific case of creating DB we start to use the general Scala API.